pkg/rancher-desktop/components/Nav.vue

```diff
@@ -15,7 +15,7 @@
     </ul>
     <template v-if="featureExtensions">
       <hr>
-      <NuxtLink to="/marketplace">
```

Comment:
I realize it's not part of this PR, but I'm wondering if the `a` tags that get rendered out of this template should be wrapped in a container element, like a `UL` or just a `DIV`. Seems to me that having a bunch of free `A` elements floating around might lead to problems in the future.

Reply:
`a` tags rendered this way are direct children of `nav`. The rendered output is semantically correct with or without `ul`:

```
<nav>
  <a>Extensions</a>
  <a>Epinio</a>
</nav>
```

vs.

```
<nav>
  <ul>
    <li>
      <a>Extensions</a>
    </li>
    <li>
      <a>Epinio</a>
    </li>
  </ul>
</nav>
```

Once we have icons for each of our Rancher Desktop navigation items, we can drop the associated `ul` altogether and simplify our CSS for navigation as well. 

Do you foresee specific issues with this approach? We could most definitely design a `nav-list` component if we want to add some more structure, but I didn't perceive much benefit from this while reviewing our navigation.

Reply:
I was just bikeshedding. This looks fine.

Reply:
Sounds good, we can follow up in another PR if needed. I'm open to changing if we think there's a benefit.